clients/manual_submarine_captain_client.py

```diff
@@ -0,0 +1,39 @@
+import json
```

Comment:
this is literally the worst way to implement the manual approach. like, the complete worst way. can you please delete your past two commits?  i will implement those changes myself.

Reply:
It's better for a feature branch to have only _one_ new feature anyway.